modules/microshift-manifests-deletion-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-manifests-deletion-overview_{context}"]
+= How manifest deletion works
```

Comment:
I might move this module to right before the _Use cases_ module. It explains a lot, about what these subdirectories are, what Microshift is doing, and so forth.

Reply:
done